README.md

```diff
@@ -385,6 +355,13 @@ The official Meteor resources page can be found [here](https://www.meteor.com/to
 
 * [We Work Meteor](https://www.weworkmeteor.com/)
 
+## Related
+
+* [Awesome Meteor Developers](https://github.com/harryadelb/awesome-meteor-developers)
+* [Awesome Blaze](https://github.com/arggh/awesome-blaze)
```

Comment:
should we move all the things related to Blaze from here to that repo and just link to it?

Reply:
Same as above. I'd rather keep them separated as not conflict things with each other.